packages/lwc-language-server/src/lwc-server.ts

```diff
@@ -141,7 +141,7 @@ export default class Server {
         if (await this.context.isLWCTemplate(doc)) {
             this.auraDataProvider.activated = false; // provide completions for lwc components in an Aura template
             this.lwcDataProvider.activated = true;
-            if (params.context?.triggerCharacter === '{') {
+            if (this.shouldProvideBindingsInHTML(params)) {
```

Comment:
Open to better naming here and for the `shouldCompleteJavascript` below, but I definitely didn't think the previous statement was clear on why we were matching on specific characters.